packages/authentication/src/definitions.ts

```diff
@@ -73,6 +73,14 @@ export interface FirebaseAuthenticationPlugin {
    * @since 0.1.0
    */
   getIdToken(options?: GetIdTokenOptions): Promise<GetIdTokenResult>;
+  /**
+   * Checks if an incoming link is a sign-in with email link suitable for signInWithEmailLink.
+   *
+   * The link sent to the user's email address must be provided.
+   *
+   * @since 0.6.0
+   */
+  isSignInWithEmailLink(options: SignInWithEmailLinkOptions): Promise<void>;
```

Comment:
```suggestion
  isSignInWithEmailLink(options: IsSignInWithEmailLinkOptions): Promise<IsSignInWithEmailLinkResult>;
```

Please name the options interface like the method and append `Options` to it: `IsSignInWithEmailLinkOptions`.
Also create a result interface.

```ts
export interface IsSignInWithEmailLinkResult {
  /**
   * Whether an incoming link is a signup with email link suitable for `signInWithEmailLink(...)`.
   */
  isSignInWithEmailLink: boolean;
}
```